### ISSUE TITLE
Please fix your acessibility issues.

### ISSUE THREAD

**Issue by lallmaier** [ant-design/ant-design#36539]
### What problem does this feature solve?

Ant has very bad Web Accessibility and a lot of the components fail WAI tests. The main problem seems to be the aria roles.
We have to use Ant for our projects and they have to be fully accessible without any errors which seems to be impossible as of right now.

### What does the proposed API look like?

Please fix those accessibility issues. thanks.

**Comment by afc163** [comment]
What are the exact problems of accessibility issues?

**Comment by lallmaier** [comment]
Wrong/Not matching ARIA roles and/or labels. Accessibility checks for the wcag2a standard find a lot of issues. For example in the menu component, where the <div> inside the <li> has the role "menu-item" instead of the <li> having it.

**Comment by lallmaier** [comment]
Since there doesn't seem to be any effort to resolve this, we are unfortunately going to stop using Ant Design, since web accessibility is of very high importance for our clients.

Feel free to close this issue :)
